.github/workflows/shiftleft.yml

```
@@ -0,0 +1,62 @@
+    - name: Setup Java JDK v8
+      uses: actions/setup-java@v3
+      with:
+        distribution: zulu
+        java-version: 8
+
+    - name: Download ShiftLeft CLI
+      run: |
+        curl https://cdn.shiftleft.io/download/sl > ${GITHUB_WORKSPACE}/sl && chmod a+rx ${GITHUB_WORKSPACE}/sl
+
+    - name: preZero Static Analysis
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:1: Double quote to prevent globbing and word splitting

(shellcheck)